core/src/epicli/cli/engine/ansible/AnsibleRunner.py

```diff
@@ -78,11 +78,12 @@ def pre_flight(self, inventory_path):
         self.ansible_command.run_playbook(inventory=inventory_path,
                                           playbook_path=self.playbook_path('common'))
 
-
     def post_flight(self, inventory_path):
         self.ansible_command.run_playbook(inventory=inventory_path,
                                           playbook_path=self.playbook_path('repository_teardown'))
 
+        self.ansible_command.run_playbook(inventory=inventory_path,
+                                          playbook_path=self.playbook_path('postflight'))
```

Comment:
I would suggest to rename this playbook to explicit "clean local kubeconf" or something similar. When I read line 83 it's quite clear what `repository_teardown` is doing, but it's not clear at all at this level what will `postflight` do. Other option in my opinion would be to move `repository_teardown` logic into `postflight` playbook.

Reply:
So the idea is to introduce similar task to preflight but just called postflight

> @sk4zuzu do we need to create `~/.kube/config` on remote? As I understand `/etc/kubernetes/admin.conf` is used everywhere.

Not really, but this is just a first PR in the chain. I will probably do what you suggest in the nex one.

Reply:
So we have a preflight task. but we dont have a postflight task. :D

I think instead of introducing more and more files in the main playbook directory I placed this in a special role that is supposed to run over all hosts in the inventory and do smaller cleanup-related tasks.

So if you look inside the role you'll see a clear distinction. I'd like to keep that. 🤗

Reply:
As you wish, mister 😝 👍